packages/hint-no-vulnerable-javascript-libraries/src/hint.ts

```diff
@@ -76,11 +75,20 @@ export default class NoVulnerableJavascriptLibrariesHint implements IHint {
         const loadSnyk = async () => {
             const oneDay = 3600000 * 24;
             const now = Date.now();
-            const snykDBPath = path.join(__dirname, 'snyk-snapshot.json');
-            let snykStat: fs.Stats;
+
+            /*
+             * We check if requestAsync exists, because if
+             * the browser extension is running this hint, then
+             * `requestAsync` will be null and we can't download
+             * the file.
+             */
+            if (!requestAsync) {
```

Comment:
We could probably look into using `new webpack.DefinePlugin({ 'process.env.webpack': JSON.stringify(true) }),`, similar to what we do in the `create` packages. This way when the env variable is defined in compile time the code is directly remove.

Reply:
@molant done!